web/client/epics/geostory.js

```diff
@@ -8,12 +8,16 @@
  */
 
 import { Observable } from 'rxjs';
```

Comment:
The epics in this files needs tests. We are usually testing the epics to verify if the actions declared are correctly dispatched. You will see the test epics use this utils https://github.com/geosolutions-it/MapStore2/blob/e5574d8bbfb9dfa95e3550d07ef0e52ac6cdbdea/web/client/epics/__tests__/epicTestUtils.js#L15.

Reply:
> The epics

I mean the epics introduced with this PR